File: C++/C++ Practice/factorial.cpp
#include<iostream>
using namespace std;
int factorial(int m);
int main(){
    int n;
    cout<<"Enter the number = ";
    cin>>n;
    int fact = factorial(n);
    cout<<"Factorial of "<<n<<" = "<<fact;
}

int factorial(int n){
    int f = 1;
    while(n>0){
        f = f * n;
        n--;
    }
    return f;
}

File: ADS/ADS_Assignments/Assignment_No_8/src/com/app/recurr/Factorial.java
package com.app.recurr;

import java.util.Scanner;

public class Factorial {
	public static int factorial(int n) {
		if(n==0) {
			return 1;
		}
		return factorial(n-1)*n;
	}
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter the number :");
		int num = sc.nextInt();
		int fact = factorial(num);
		System.out.println(fact);
	}

}
